### PR TITLE
Adding cert-manager expiration check example to opa

### PR DESCRIPTION
**Pull request by transient1**
## Description
Adds an example for a `cert-manager` cert expiration check. 

## Checklist

* [ ] I have updated the CHANGELOG of any plugins updated
* [ ] I have updated the version file of any plugins updated
* [ ] I have added tests wherever appropriate
* [ ] I have labeled this PR with a label per plugin updated

### CODE REVIEW

**Review by rbren — APPROVED** [review]
🎉 Thanks Stevie!

Would love to get @baderbuddy's thoughts too

**Comment by rbren** [comment]
You'll need to bump  plugins/opa/version.txt and  plugins/opa/CHANGELOG.md

File: plugins/opa/examples/cert-expiration/deployments.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+clusters: null
+output: null
+parameters:
+  days: 60
+runEnvironments: null
+targets:
+  - apiGroups:
+      - ""
+    kinds:
+      - Secret
```
Comment (baderbuddy) [inline plugins/opa/examples/cert-expiration/deployments.yaml:10]: We should probably change the name of this file, either `certs` or `secrets` or something would work.

**Review by baderbuddy — COMMENTED** [review]
Sorry did you miss a commit? It looks like this is the helm chart is fresh rego

File: plugins/opa/examples/cert-expiration/deployments.yaml (hunk 1 above)
Comment (transient1) [reply plugins/opa/examples/cert-expiration/deployments.yaml:10]: Changed to certs.yaml

**Pull request merged**